### PR TITLE
Make Fedora VMI small and timeouts the same

### PR DESCRIPTION
**Pull request by petrkotas**
**What this PR does / why we need it**:
Synchronize the wait period for migration.
Make the Fedora VM as small as possible, where applicable.

**Release note**:

```release-note
NONE
```

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/kubevirt/2570)

### CODE REVIEW

**Comment by petrkotas** [comment]
Hi @slintes, you are of course right. The constants are nicer. Addressed that.
Also yes, once this one made it in, the other two can be closed.

File: tests/migration_test.go [hunk 1]
```diff
@@ -1577,7 +1582,7 @@ func fedoraVMIWithEvictionStrategy() *v1.VirtualMachineInstance {
 	vmi := tests.NewRandomFedoraVMIWitGuestAgent()
 	strategy := v1.EvictionStrategyLiveMigrate
 	vmi.Spec.EvictionStrategy = &strategy
-	vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("1Gi")
+	vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("256M")
```
Comment (slintes) [inline tests/migration_test.go]: you missed this one for fedoraVMSize

Comment (petrkotas) [reply tests/migration_test.go]: And its done :). Thanks for spotting this bad boy.

**Comment by slintes** [comment]
previous test runs looked good already, but let's wait for latest results before merging

/lgtm
/approve

**Pull request merged**